Partition (WBP)/Functions/V57033H.VT157033.fun.yml

```diff
@@ -139,6 +150,7 @@
     #F1570331 #F1570331_VARCHAR #F1570338 #F1570338_NCHAR #F1570339 #F1570339_NVARCHAR #F157033A #F157033A_ALPHA #F157033H #F157033H_CHAR #F157033U #F157033U_SUNI := #LF_Test_Data
     
     Insert Fields(*ALL) To_File(VTL157033H) Io_Status(#this_rc) Io_Error(*NEXT) Val_Error(*NEXT)
+    Insert Fields(*ALL) To_File(VTL157033I) Io_Status(#this_rc) Io_Error(*NEXT) Val_Error(*NEXT)
```

Comment:
Each table needs separate RC testing.

Reply:
Ok..I will fix this ,push and then move to 'J'

Reply:
agreed